build.gradle

```diff
@@ -77,6 +77,9 @@ configure(javaProjects) {
             // AssertJ
             dependency "org.assertj:assertj-core:${ext.versionOf('assertj')}"
 
+            // Completable-futures
+            dependency "com.spotify:completable-futures:${ext.versionOf('completable-futures')}"
```

Comment:
Sounds reasonable. I would like to leave this in this PR (and want to fix in next PR)